src/effects/effectparameter.h

```diff
@@ -39,6 +39,7 @@ class EffectParameter : public QObject {
     ///////////////////////////////////////////////////////////////////////////
 
     EffectManifestParameter::LinkType getDefaultLinkType() const;
+    EffectManifestParameter::LinkInversion getDefaultLinkInversion() const;
```

Comment:
OK, thanks for the quote. I'm not really convinced that's the best design, but it does make sense to follow the existing code base if there's a valid reason to do so.